samples/nrf9160/at_client/README.rst

```diff
@@ -37,24 +37,24 @@ Building and running
 Connecting to the nRF9160 DK board with LTE Link Monitor
 ========================================================
 
-To connect to the nRF9160 DK board with LTE Link Monitor, perform the following steps:
-
-#. Launch the LTE Link Monitor after programming the AT Client sample onto the nRF9160 DK board.
-#. Make sure that **Automatic requests** is enabled in LTE Link Monitor.
-#. Connect the nRF9160 DK board to the PC with a USB cable.
-#. Power on the nRF9160 DK board.
-#. Click **Select Device** and select the particular board device entry from the drop-down list in the LTE Link Monitor.
-#. Observe that initially the command :command:`AT+CFUN?` is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
-#. Observe that the LTE Link Monitor terminal display also shows :command:`AT+CFUN=1` indicating that the modem has changed to the normal mode.
-#. Press the reset button on the nRF9160 DK to reboot the board and start the AT Client sample.
+See :ref:`lte_connect` for connecting to the nRF9160 DK board with LTE Link Monitor.
+While you connect to the nRF9160 DK board, observe that the following communication handshakes occur between the external computer or MCU and the nRF9160 modem in the LTE Link Monitor display terminal:
 
+1. Initially the command AT+CFUN? is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode.
```

Comment:
Should we move these steps down to line 58?

Reply:
Moved this under Testing section.